tests/manifold/spherical_manifold_03.output

```diff
@@ -1,3 +1,3 @@
 
-DEAL::Testing dim 2, spacedim 2
-DEAL::Testing dim 3, spacedim 3
+DEAL::1.57080 0.00000 0.00000
+DEAL::0.00000 1.57080 0.00000
```

Comment:
Ah, excellent. By the way, no need to *replace* tests by something new -- it's totally fine to just *add more* tests. We cannot have too many tests.